gui/default/assets/css/overrides.css

```diff
@@ -539,6 +539,20 @@ ul.three-columns li, ul.two-columns li {
     cursor: not-allowed;
 }
 
+@media (prefers-color-scheme: dark) {
```

Comment:
I've just spotted that these overrides are also applying to the light theme (that is, if you use the light theme with a system dark mode, the media query override from default applies).  Should I move these to default's theme.css instead of overrides.css?

Reply:
Scratch that, I've just seen how the default theme pulls in dark and light depending on the mode, so I can just drop this entirely now.